compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/CollectionStubMethodLowering.kt

```diff
@@ -136,12 +136,24 @@ private class CollectionStubMethodLowering(val context: JvmBackendContext) : Cla
         return mutableClass.typeParameters.map { it.symbol }.zip(readOnlyClassTypeArguments).toMap()
     }
 
+    // Preserve old backend's logic to generate stubs for special cases where a mutable method
+    // have same jvm signature with an immutable method.
+    private val specialSignatures = setOf(
```

Comment:
Could you please name this variable something more specific and reflecting its purpose?

For example, `specialCaseStubSignaturesForOldBackend`.

And also please mention the issue [KT-36724](https://youtrack.jetbrains.com/issue/KT-36724) in the comment. We'd like to fix this issue in the old backend at some point.